### PR TITLE
fix runners count and add defaults to use all cores

### PR DESCRIPTION
**Pull request by Jomik**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -6,11 +6,5 @@ bots:
   simplebot: screepsbot-zeswarm
 
 serverOptions:
-  # The number of parallel runner threads in which player scripts are executed.
-  # Don't set this option greater than the number of your physical CPU cores.
-  runnerThreads: 4
-  # The number of room processor worker processes to launch.
-  # Don't set this option greater than the number of your physical CPU cores.
-  processorsCnt: 2
```
Comment (o4kapuk) [inline config.yml:14]: Why is that removed?

File: screeps-start.js [hunk 2]
```diff
@@ -125,8 +126,8 @@ const writeModsConfiguration = () => {
 
 // Map from camelCase to snake_case
 const ServerConfigMap = {
-  runnersThreads: "runner_threads",
-  processorsCnt: "processors_cnt",
+  runnerCount: "runners_threads",
```
Comment (o4kapuk) [inline screeps-start.js]: runners_threads is not a great name to determine the runners **count**. We should get rid of runner_threads completely

File: config.yml (hunk 1 above)
Comment (Jomik) [reply config.yml:14]: To not override the defaults specified in `screeps-start.js`, which does the `#cores - 1` for runners, and `#cores` for processors.
People tend to just copy this and leave it, so better to remove it.
I think I will add a section in the wiki for these Server Options though?

File: screeps-start.js (hunk 2 above)
Comment (Jomik) [reply screeps-start.js]: Done, was a miss on my end. It will be runner count and threads are gone.

File: screeps-start.js [hunk 3]
```diff
@@ -138,9 +139,12 @@ const start = async () => {
   writeModsConfiguration();
 
   const screeps = require("@screeps/launcher");
+  const cores = os.cpus().length;
```
Comment (o4kapuk) [inline screeps-start.js:142]: https://nodejs.org/api/os.html#oscpus
> used to get information about each **logical** CPU core of the computer. 

We need physical cores here
